src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -298,6 +298,8 @@ public IEnumerable<WorkspaceModel> Workspaces
         {
```

Comment:
Please revert this file since it has no real change:

    git checkout master src/DynamoCore/Models/DynamoModel.cs